code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1283,22 +1281,163 @@
 		to_chat(affected_mob, span_notice("Ah, what was that? You thought you heard something..."))
 		affected_mob.adjust_confusion(5 SECONDS)
 
-/datum/reagent/toxin/hunterspider
+// Most spiders inject 3u of venom per bite (with the exception of assassain that injects 5u)
+/datum/reagent/toxin/spider
 	name = "Spider Toxin"
-	description = "A toxic chemical produced by spiders to weaken prey."
-	health_required = 40
+	description = "Hello! I am a bugged reagent. Please report me for my crimes. Thank you!!"
+	toxpwr = 0
 	liver_damage_multiplier = 0
+	chemical_flags = REAGENT_NO_RANDOM_RECIPE
 
-/datum/reagent/toxin/viperspider
-	name = "Viper Spider Toxin"
-	toxpwr = 5
-	description = "An extremely toxic chemical produced by the rare viper spider. Brings their prey to the brink of death and causes hallucinations."
-	health_required = 10
-	liver_damage_multiplier = 0
+/datum/reagent/toxin/spider/paralysis
+	name = "Paralysis Spider Toxin"
+	description = "A toxic chemical produced by spiders to incapacitate prey."
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	toxpwr = 0.1
+	liver_damage_multiplier = 1
+	var/list/parlyzed_limbs = list()
+	var/list/unaffected_limbs = list(
+		TRAIT_PARALYSIS_L_ARM = "left arm",
+		TRAIT_PARALYSIS_R_ARM = "right arm",
+		TRAIT_PARALYSIS_L_LEG = "left leg",
+		TRAIT_PARALYSIS_R_LEG = "right leg",
+	)
```

Comment:
I believe you need to implement `on_merge` for this as well, or use `data`

Reply:
So I checked the comments for both of these:

https://github.com/tgstation/tgstation/blob/22337bcfc93f44895d9cb89e26654e0e7e516e91/code/modules/reagents/chemistry/reagents.dm#L17-L18

https://github.com/tgstation/tgstation/blob/22337bcfc93f44895d9cb89e26654e0e7e516e91/code/modules/reagents/chemistry/reagents.dm#L221-L223

I don't believe we should use either. 

- `on_merge` we don't care about if the same poison gets mixed together twice since we want the raw volume only. This is used in other places to calculate the color or purity of the new chem which we don't care about.
- `data` keeps track of information concurrently as the reagent is split into two. In the rare chance that happens, we want all our paralysis lists to be reset to the original state. (which my current code should do) That way if someone has 100u of spider poison, and they get gibbed and their meat is given to another person (let's say 5u) it doesn't trigger full paralysis.